whyis/materialsmine/nanomine.ttl

```diff
@@ -0,0 +1,3240 @@
+@prefix : <http://nanomine.org/> .
+@prefix bibo: <http://purl.org/ontology/bibo/> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix nanomine: <http://nanomine.org/ns/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sio: <http://semanticscience.org/resource/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+<<<<<<< HEAD
```

Comment:
It seems like there are some git merge conflict markers still throughout this file

Reply:
I've deleted the file, since resolving all those doesn't make sense (it's big and autogenerated).